ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -257,6 +263,42 @@ export default class TestDataModal extends React.Component {
     this.closeModal();
   };
 
+  selectableTestClassName = test => {
+    if (this.hasDifferentUnit(test)) {
+      return 'bg-warning';
+    }
+    return '';
+  };
+
+  selectableTestTitle = test => {
+    if (this.hasDifferentUnit(test)) {
+      const unit = test.measurementUnit;
+      return `Warning: ${this.getOriginalTestName(
+        test,
+      )} has a different measurement unit (${unit}) `;
+    }
+    return this.getOriginalTestName(test);
+  };
+
+  hasDifferentUnit = test => {
```

Comment:
I think that this is more readable:
```js
  hasDifferentUnit = test => {
    const { plottedUnits } = this.props;
    const { selectedUnits } = this.state;
    const unit = test.measurementUnit || null;

    const condition1 = plottedUnits.size && !plottedUnits.has(unit);
    const condition2 = selectedUnits.size >= 2; // already a mismatch;
    const condition3 = (selectedUnits.size && !selectedUnits.has(unit);

    return condition1 || condition2 || condition3;
  };
```
Please give appropriate names for `condition1, condition2, condition3` variables.

Reply:
Agree.